qa/L0_backend_python/argument_validation/test.sh

```diff
@@ -75,4 +75,8 @@ else
     echo -e "\n***\n*** Argument validation test PASSED. \n***"
 fi
 
+# Collect all logs and core dumps and copy them to an upper-level directory for
+# proper capture on the CI.
+cp *.*log* core* ../ || true
```

Comment:
Nice updates.

Can this be some function in ../common/util.sh so it only needs to be updated in one spot?

Reply:
Great idea! Updated.